test/electricity/electromotive_force_of_induction_in_rotating_ce_of_induction_in_rotating_circuit_test.py

```diff
@@ -0,0 +1,81 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy import pi
+from symplyphysics import (assert_equal, units, Quantity, errors)
+from symplyphysics.laws.electricity import electromotive_force_of_induction_in_rotating_ce_of_induction_in_rotating_circuit as electromotive_force_law
```

Comment:
`electromotive_force_of_induction_in_rotating_ce_of_induction_in_rotating_circuit`
What's up with the name?